WooCommerce/WooCommerceUITests/Flows/MockDataReader.swift

```diff
@@ -51,4 +51,24 @@ class GetMocks {
 
         return updatedData
     }
+
+    static func readOrdersData() throws -> [OrderData] {
+        let originalData = try JSONDecoder().decode(OrderMock.self, from: self.getMockData(test: OrdersTests.self, filename: "orders_any"))
+        var updatedData = originalData.response.jsonBody.data
+
+        for index in 0..<updatedData.count {
+                    var total = updatedData[index].total
+                    print("total is: \(total)")
+                    print("total after converted: \(Double(total) ?? 0)")
+                    let numberFormatter = NumberFormatter()
+                    numberFormatter.numberStyle = .decimal
+
+                    let formattedNumber = numberFormatter.string(from: NSNumber(value: Double(total)!))
+                    print("formatted total is: \(formattedNumber)")
+                    updatedData[index].total = formattedNumber!
+                }
```

Comment:
It looks like the indentation of this block is two tabs deeper than it should be:

```suggestion
            let total = updatedData[index].total
            print("total is: \(total)")
            print("total after converted: \(Double(total) ?? 0)")
            let numberFormatter = NumberFormatter()
            numberFormatter.numberStyle = .decimal

            let formattedNumber = numberFormatter.string(from: NSNumber(value: Double(total)!))
            print("formatted total is: \(formattedNumber)")
            updatedData[index].total = formattedNumber!
        }
```

(you can also notice that I've changed the type for `total` to a constant, since it's never changed in the method)

Reply:
Good call; I had initially made that `total` variable when I was figuring out how I'd account for formatting differences between mocks and the UI, but ended up not needing it.
Also fixed the indentation, thanks!